exercises/practice/darts/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/
+
+if [ ! -f $SCRIPT_DIR/bin/cobolcheck ]; then
+    echo "Cobolcheck not found, try to fetch it."
+    cd $SCRIPT_DIR/bin/
+    ./fetch-cobolcheck
+fi
+cd $SCRIPT_DIR
+./bin/cobolcheck -p leap
```

Comment:
Hey @axtens, copy the test.sh and test.ps1 from one of the other exercises. @0xE282B0 changed them so that we don't need to write the name of the exercise there.